### PR TITLE
Bugfix/label event count resolution

### LINKED ISSUE #91: No configuration to resolve 'date' from complex datasets
The `0.2.0` release appears to have removed documented support for a custom function to resolve the date from a more complex data set. 

For example, after installing `EventDrops` from npm, I tried to make an example using the code from the `README` with data of the form
 ```
var data = [
  { name: "http requests", data: [{date: new Date('2014/09/15 13:24:54'), foo: 'bar1'}, {date: new Date('2014/09/15 13:25:03'), foo: 'bar2'}, {date: new Date('2014/09/15 13:25:05'), foo: 'bar1'}, ...] },
  { name: "SQL queries", data: [{date: new Date('2014/09/15 13:24:57'), foo: 'bar4'}, {date: new Date('2014/09/15 13:25:04'), foo: 'bar6'}, {date: new Date('2014/09/15 13:25:04'), foo: 'bar2'}, ...] }
];
```

The docs say that I should be able to configure the chart using `.date(d => d.date)`. However, `date` was at some point removed as a configurable parameter from default `config.js` which means that this function doesn't work and errors out when it is used. 

I was wondering if the documentation on GitHub should be synchronized with the `0.2.0` release, and if there was a new way to use this sort of functionality

### PR DESCRIPTION
**Pull request by twisty-n**
Hi guys, 

This addresses an issue when using a custom date resolver. The row labels weren't displaying the count of associated events due to the filter not using the config resolver. 

Fixes #91 (in regards to some documentation quirks)

`width()` was removed from the examples because it is no longer a supported

### CODE REVIEW

**Comment by jpetitcolas** [comment]
Thanks for your PR! However, can you add some unit tests on `filterData` to ensure problem won't happen again? :)

**Comment by twisty-n** [comment]
No worries. I will do so :)

**Comment by twisty-n** [comment]
@jpetitcolas Update with the unit tests :)

**Comment by deevus** [comment]
Bump

File: lib/filterData.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-export default function filterData(data = [], scale) {
+export default function filterData(data = [], scale, config) {
```
Comment (jpetitcolas) [inline lib/filterData.js]: Can you instead just pass the `date` function?

**Comment by jpetitcolas** [comment]
Megeable once the last comment is done.

**Comment by jpetitcolas** [comment]
Thanks! :)

**Pull request merged**